src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ReferencesPage/ImplicitUsingsValueProvider.cs

```diff
@@ -0,0 +1,200 @@
+// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[ExportInterceptingPropertyValueProvider("ImplicitUsingsEditor", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+[AppliesTo(ProjectCapability.CSharp)]
```

Comment:
Note that `AppliesTo` attributes are not currently honored when importing interceptors, so make sure only C# projects use a property by the name of "ImplicitUsingsEditor".

It is still valuable to have to `AppliesTo` attribute in preparation for the day we fix this.